Library/Formula/saltstack.rb

```diff
@@ -12,7 +15,7 @@ class Saltstack < Formula
   head "https://github.com/saltstack/salt.git", :branch => "develop", :shallow => false
 
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "swig" => :build
+  depends_on "swig304" => :build
```

Comment:
I know that M2Crypto is unmaintained.

Now how to obtain last saltstack last dev or last stable version to build in OSX ?



-- 
Julien Lavergne
Sent with Airmail

On April 14, 2015 at 4:36:06 AM, tim smith (notifications@github.com) wrote:

In Library/Formula/saltstack.rb:

> @@ -12,7 +15,7 @@ class Saltstack < Formula
>    head "https://github.com/saltstack/salt.git", :branch => "develop", :shallow => false
>   
>    depends_on :python if MacOS.version <= :snow_leopard
> -  depends_on "swig" => :build
> +  depends_on "swig304" => :build
I ran into this trying to bump saltstack earlier. M2Crypto is broken. It can be made to work with newer swigs but requires patching and I don't think anything's been committed anywhere since M2Crypto seems undermaintained. Some context here: swig/swig#344 and here: martinpaljak/M2Crypto#60 (comment)

—
Reply to this email directly or view it on GitHub.